From 72530ecfdd8cc108a779a1e0b2c500535220ca3e Mon Sep 17 00:00:00 2001
From: mark <mark.appledeveloper@gmail.com>
Date: Mon, 1 Jun 2020 14:27:26 +0800
Subject: [PATCH 1/5] fix the ButtonBarPager layout size [by
 69ffa5c61da90ad860f2be12257593da93a131b5] fix XLPager in viewDidAppear of
 ButtonBarPagerTabStripViewController [by
 dffbdeb10a8ae4e1f3541668ba0cf02a3ed13c96] fix XLPager reload 2 times [by
 df33b9c755369c9ff2726928b66de877612ae97e]

---
 Sources/ButtonBarPagerTabStripViewController.swift | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/Sources/ButtonBarPagerTabStripViewController.swift b/Sources/ButtonBarPagerTabStripViewController.swift
index 36e008d..5fa6fd4 100644
--- a/Sources/ButtonBarPagerTabStripViewController.swift
+++ b/Sources/ButtonBarPagerTabStripViewController.swift
@@ -165,6 +165,12 @@ open class ButtonBarPagerTabStripViewController: PagerTabStripViewController, Pa
         buttonBarView.layoutIfNeeded()
     }
 
+    open override func viewDidAppear(_ animated: Bool) {
+        view.setNeedsLayout()
+        view.layoutIfNeeded()
+        super.viewDidAppear(animated)
+    }
+
     open override func viewDidLayoutSubviews() {
         super.viewDidLayoutSubviews()
 
-- 
2.21.1 (Apple Git-122.3)

